frontend/express/public/javascripts/countly/vue/components/sidebar.js

```diff
@@ -141,6 +143,18 @@
                     document.body.appendChild(logoutForm);
                     logoutForm.submit();
                     document.body.removeChild(logoutForm);
+                },
+                onCopy: function() {
+                    CountlyHelpers.notify({
+                        message: this.successmessage,
```

Comment:
"m" is lower case here, while it was defined as successMessage. Please pay more attention to such details :slightly_smiling_face: